File: Scenario-5/Python/main.py
# Scenario 5: API Integration
# You can run the code by 'pthon main.py' command in terminal.

# Task1: Integrate restfull API into your application
# Task2: Beatufy the output of the API response


import requests

def get_object_data():
    url = "https://api.restful-api.dev/objects/7"
    return "Sample output"

def main():
    print(get_object_data())  # Should print the object data from the API

if __name__ == "__main__":
    main()

File: Scenario-5/Javascript/main.js
// Scenario 5: API Integration
// You can run the code by 'dotnet run' command in terminal.

// Task1: Integrate RESTful API into your application
// Task2: Beautify the output of the API response

async function getObjectData() {
    const url = 'https://api.restful-api.dev/objects/7';
    return 'Sample response';
}

async function main() {
    console.log(await getObjectData());  // Should print the object data from the API
}

main();
